src/System.Text.Json/tests/JsonNumberTestData.cs

```diff
@@ -204,7 +207,7 @@ static JsonNumberTestData()
             }
 
             builder.Append("\"intEnd\": 0}");
-            #endregion
+#endregion
```

Comment:
nit: revert indentation change

Reply:
You should push for https://github.com/dotnet/roslyn/issues/12305 to be resolved so you don't need to fight the IDE/auto-formatting on this.

Reply:
Or just delete all the \#regions :smile:

Reply:
> so you don't need to fight the IDE/auto-formatting on this.

Given this end region is the only one that changed (and not the rest), I don't think the IDE is fighting the formatting on this.

I just tried Ctrl+K/Ctrl+D, and it doesn't un-indent this (in-fact it re-indents it for me). Can you explain why the IDE only re-formatted the last one?

Reply:
> I just tried Ctrl+K/Ctrl+D, and it doesn't un-indent this (in-fact it re-indents it for me). Can you explain why the IDE only re-formatted the last one?

Because it comes down to the root editorconfig not having a style preference for this, and it resulting back to whatever behavior VS has set.